app/components/NavLinks.tsx

```diff
@@ -8,87 +8,98 @@ import { useRouterWithOptimisticPathname } from "../hooks/useOptimisticRouter";
 export const Navbar = () => {
   const router = useRouterWithOptimisticPathname();
   const pathname = usePathname();
-  const role = useAppSelector((state) => state.auth.role);
-  const isAdmin = role === "admin";
-
+  const role = useAppSelector(state => state.auth.role)
+  const isAdmin = role === 'admin';
   const handleClick = () => {
-    Cookies.remove("access_token");
-    router.push("/login");
-  };
-
-  let navText = "SF Social Club";
-
-  if (isAdmin) {
-    navText += " Admin Page";
+    Cookies.remove('access_token')
```

Comment:
Non-blocker cuz we iterating fast, but we should aim to take out all of these to a constant or enum to reuse. Same with 'admin' | 'member' etc. Lets try to take advantage of typescript where we can